### PR TITLE
Enhancement/filters

### PR DESCRIPTION
**Pull request by arbreurkes**
Made sure all filters work on the phylo tree.

### CODE REVIEW

**Comment by arbreurkes** [comment]
Will fix merge conflicts during tomorrow's merging session!

File: src/main/java/core/Filtering.java [hunk 1]
```diff
@@ -0,0 +1,354 @@
...
+/**
+ * Created by Arthur on 6/6/16.
+ */
+public class Filtering {
+    private List<Filter> filters;
+    private List<Genome> selectedGenomes;
+    private Predicate<Genome> predicate;
+    private List<Predicate<Genome>> predicates;
+
+    public Filtering() {
```
Comment (ghost) [inline src/main/java/core/Filtering.java:25]: Public methods require JavaDoc. CheckStyle & co will warn for this.

Comment (tieswesterborg) [inline src/main/java/core/Filtering.java:25]: If you fix this and fix your merge conflicts, PR is ready to merge.

**Comment by tieswesterborg** [comment]
No remarks.

**Pull request merged**